config/chime_science_run_recv_hfb.yaml

```diff
@@ -37,7 +37,7 @@ hfb_pool:
 
 hfbbuf_10s:
   kotekan_buffer: hfb
-  num_frames: buffer_depth * 6
+  num_frames: buffer_depth * 32
```

Comment:
I would just give this is 2048 directly.   In case we have more components and want to change their buffer depths.   This receiver buffer is a special case.